docs/source/tutorials/low-latency.rst

```diff
@@ -113,6 +113,19 @@ branch of the DepthAI. This will pass pointers (at XLink level) to cv2.Mat inste
 so performance improvement would depend on the image sizes you are using.
 (Note: API differs and not all functionality is available as is on the `message_zero_copy` branch)
 
+PoE latency
+###########
+
+On PoE, the latency can vary quite a bit due to a number of factors:
```

Comment:
Let's mention these advanced settings for network gurus: https://github.com/luxonis/depthai-core/pull/428